schema/ast.go

```diff
@@ -1,5 +1,7 @@
 package schema
 
+import "github.com/k0kubun/sqldef/sqlparser"
```

Comment:
The whole point of `github.com/k0kubun/sqldef/schema` package is to decouple `sqlparser` from other sqldef implementations. Thus this file should not import `sqlparser`. Could you consider not to use `sqlparser.Permissive` in this package?

Reply:
OK, I fixed it.
https://github.com/k0kubun/sqldef/pull/77/commits/a2e003ba5005cc02d9f317847a4b76a4310a4978